o1vm/src/pickles/prover.rs

```diff
@@ -225,7 +230,9 @@ where
         quotient
     };
 
-    let _t_comm = srs.commit_non_hiding(&quotient_poly, 7);
+    let t_comm = srs.commit_non_hiding(&quotient_poly, DEGREE_QUOTIENT_POLYNOMIAL as usize);
```

Comment:
Not a part of this PR, but do we want to do chunking ?
We do not have any limitation on the SRS